### DISCUSSION TITLE
Clone repository on small responsive screen

### DISCUSSION THREAD

**Discussion by disgone** [community/community#33397] [New to GitHub]
I may simply be missing or overlooking this, but is there a clone / codespaces option when viewing a repository via Github on the web @ less than 750px width?  I typically split my monitor into zones and one zone happens to fall into this smaller category and I noticed I couldn't find the option like in larger sizes (the green "code" callout).

>750px
![image](https://user-images.githubusercontent.com/56998/190888435-89eb53ae-8873-495f-87a6-0fe2451b9a9a.png)

<750px
![image](https://user-images.githubusercontent.com/56998/190888464-c191c25f-c594-4b8d-8a2a-e9a65be9295e.png)

**Comment by radar-blip** [comment]
If I can remember right, the green 'code' button used to say 'clone'. 
Dont know if that answers your question.

**Reply by disgone** [reply]
Thanks -- unfortunately that's not exactly what I'm after.

The clone and codespaces shortcuts _are_ located under than green "Code" button in larger screen sizes -- however, if you scale the viewport down past that 750px responsive breakpoint the code button is hidden and as far as I can tell there's no alternative.  Since cloning is so fundamental this seems silly to completely remove, so I figured I was just overlooking where this may get shifted to.

**Reply by radar-blip** [reply]
Yes I see now. The button is completely gone?
Sorry, I dont have an answer for you.
